src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html

```diff
@@ -14,7 +14,11 @@
     <nav class="navbar navbar-expand-lg bg-warning navbar-dark py-1">
         <div class="container-fluid">
           <div class="col-md-1 text-center d-none d-md-block">
+<<<<<<< Updated upstream:src/main/VirtuDoc - Records Pages/Front-End/VirtuDoc - Patient Records Page/index.html
```

Comment:
Looks like there's some leftovers from a merge conflict here on L17,19,21.

Reply:
They are removed now, I think the changes went through?